wiggum_app/static/js/distanceMatrixHeatmap.js

```diff
@@ -345,7 +345,10 @@ function distanceMatrixHeatmap(options) {
 /**
  * Draw heatmap legend
  *
- * @param overviewLegendTypes - overview legend types.
+ * @param overviewLegendTypes - Legend types for overview distance matrices. 
+ * 								It is generated by extracting the unique trend types
+ * 								from all distance matrix heatmaps.
+ * 								Overview legend type can be binary or continuous or both.
```

Comment:
```suggestion
 * 		It is generated by extracting the unique trend types
 * 		from all distance matrix heatmaps.
 * 		Overview legend type can be binary or continuous or both.
```

unless that many tabs is standard for some reason